pkg/i18n/english.go

```diff
@@ -851,6 +863,18 @@ func EnglishTranslationSet() TranslationSet {
 		CantCheckoutBranchWhilePulling:      "You cannot checkout another branch while pulling the current branch",
 		CantPullOrPushSameBranchTwice:       "You cannot push or pull a branch while it is already being pushed or pulled",
 		FileFilter:                          "Filter files by status",
+		CopyToClipboardMenu:                 "Copy to clipboard",
+		CopyFileName:                        "File name",
+		CopyFilePath:                        "Path",
+		CopyFileDiffTooltip:                 "If there are staged items, this command considers only them. Otherwise, it considers all the unstaged ones.",
+		CopySelectedDiff:                    "Diff of selected file",
+		CopyAllFilesDiff:                    "Diff of all files",
+		NoContentToCopyError:                "Nothing to copy",
+		FileCopyNoItemSelectedError:         "You need to select a file/directory",
```

Comment:
I'm not sure it's worth distinguishing between the two cases. The second one is actually a bit misleading when you select an untracked file, or a directory with only untracked files, because the user _did_ select a file, just not a tracked or staged one. Personally I feel it's good enough to show "Nothing to copy" in all cases.